Sources/ArcGISToolkit/Components/FeatureFormView/EmbeddedFeatureFormView.swift

```diff
@@ -24,17 +24,27 @@ struct EmbeddedFeatureFormView: View {
     
     var body: some View {
         if let embeddedFeatureFormViewModel {
-            ScrollViewReader { scrollViewProxy in
-                ScrollView {
-                    VStack(alignment: .leading) {
-                        ForEach(embeddedFeatureFormViewModel.visibleElements, id: \.self) { element in
-                            makeElement(element)
+            ScrollViewReader { scrollView in
+                Group {
+                    if UserDefaults.standard.testCase != nil {
```

Comment:
UI tests are run in debug mode, right? If so, I would strongly suggest only running this check in debug mode:

```swift
                    let form = Form {
                        makeVisibleElements()
                    }
#if RELEASE
                    form
#else
                    if UserDefaults.standard.testCase != nil {
                        ...
                    } else {
                        form
                    }
#endif
```